addons/mail/static/src/core/common/thread_model.js

```diff
@@ -638,29 +651,34 @@ export class Thread extends Record {
         ) {
             return;
         }
+        this.isLoaded = false;
+        this.scrollTop = undefined;
+        let hasError = false;
         try {
-            this.isLoaded = false;
-            this.scrollTop = undefined;
             this.messages = await this.fetchMessages({ around: messageId });
```

Comment:
Assigning messages to thread is already too much in a try except targetted at rpc